sources/packages/web/src/views/student/StudentAccountActivity.vue

```diff
@@ -20,32 +23,19 @@
   </student-page-container>
 </template>
 <script lang="ts">
-import { computed, onMounted, defineComponent } from "vue";
+import { defineComponent } from "vue";
 import { StudentRoutesConst } from "@/constants/routes/RouteConstants";
-import { StudentRestriction } from "@/store/modules/student/student";
-import { useStudentStore } from "@/composables";
 import StudentScholasticStandingLimitedHistory from "@/components/common/students/StudentScholasticStandingLimitedHistory.vue";
-
-interface StudentAccountActivityFormModel {
-  restrictions: StudentRestriction[];
-}
+import StudentAccountRestrictions from "@/components/common/students/StudentAccountRestrictions.vue";
 
 export default defineComponent({
-  components: { StudentScholasticStandingLimitedHistory },
+  components: {
+    StudentAccountRestrictions,
+    StudentScholasticStandingLimitedHistory,
+  },
   setup() {
-    const { activeRestrictions, updateRestrictions } = useStudentStore();
-
-    const initialData = computed<StudentAccountActivityFormModel>(() => {
-      return {
-        restrictions: activeRestrictions.value,
-      };
-    });
-    onMounted(async () => {
-      await updateRestrictions();
-    });
     return {
       StudentRoutesConst,
-      initialData,
```

Comment:
Please remove the intital data to form container assignment. 
![image](https://github.com/user-attachments/assets/db75588d-b23b-46b5-b506-2bd0c5f7e451)